secbox

```diff
@@ -241,13 +241,13 @@ suse_internal_network() {
 
 secbox_instances_using_nfs() {
     local _instances=0
-    _instances=$(ps -xo pid,cmd | grep -v grep | grep -v $$ | grep -E "secbox.* --nfs" | wc -l)
+    _instances=$(pgrep --count --full "secbox.* --nfs")
```

Comment:
In order to make it works with `pgrep` the following check needs to be changed to `-le 3`. Can you please add this?

https://github.com/StayPirate/secbox/blob/983fb1db89ed372a3db1d5402f1d4422581082f5/secbox#L115